GithubUpdateCheck/GithubUpdateCheck.cs

```diff
@@ -289,8 +289,16 @@ private string NormalizeVersionString(string version)
                     return NormalizeVersionStringBoolean(version);
             }
         }
-
-
+        /// <summary>
+        /// Returns the latest version on github
+        /// </summary>
+        /// <returns>string</returns>
+        public string Version()
+        {
+            string resolved = GetResponseUrl(githubUrl + Username + "/" + Repository + latestVersionString);
+            string version = NormalizeVersionStringIncremental(resolved);
```

Comment:
Can you use the `NormalizeVersionString` instead of `NormalizeVersionStringIncremental`?
That way the lib respects the users choice, made in the constructor (incremental vs. boolean)

Reply:
Oh yes I updated it, I must have entered the wrong function I meant to call NormalizedVersionString()